packages/recomponents/src/components/r-select/r-select.story.js

```diff
@@ -516,4 +518,71 @@ storiesOf('Components.Select', module)
         `,
     }), {
         notes: {markdown},
+    })
+    .add('Validation', () => ({
+        data() {
+            return {
+                value: [],
+                options: [{
+                    label: 'Cat',
+                    value: 'cat',
+                }, {
+                    label: 'Doge',
+                    value: 'doge',
+                }, {
+                    label: 'Hamster',
+                    value: 'hamster',
+                }, {
+                    label: 'Chair',
+                    value: 'chair',
+                }],
+            };
+        },
+        props: {
+            multiple: {
+                default: boolean('Multiple', true),
+            },
+            tagPlaceholder: {
+                default: text('Tag placeholder', 'Press enter to create a tag'),
+            },
+            tagPosition: {
+                default: select('Tag position', {top: 'top', bottom: 'bottom'}, 'top'),
+            },
+            taggable: {
+                default: boolean('Taggable', true),
+            },
+        },
+        methods: {
+            select: action('select'),
+            tag: action('tag'),
+        },
+        computed: {
+            label() {
+                return `Select only pets: ${this.$v.$invalid ? 'invalid' : 'valid'}`;
+            },
+        },
+        mixins: [validationMixin],
+        validations: {
+            value: {
+                required,
+                pet: helpers.regex('alpha', /(cat|hamster|doge)/),
```

Comment:
will it work for multiple? shouldn't we use `$each`?

Reply:
Nope, but this was not the goal - for multiple we can create one more story or update this one.